templates/js/custom-copilot-basic/README.md.tpl

```diff
@@ -0,0 +1,101 @@
+# Overview of the AI Chat Bot template
+
```

Comment:
Title: `Overview of the Basic AI Chat Bot template`, the title needs to align with what's shown in the UX

Reply:
Good point. Updated.